services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/android/attestation/signature/ProdSignatureVerificationStrategy.java

```diff
@@ -3,8 +3,10 @@
 import com.google.api.client.json.webtoken.JsonWebSignature;
 import java.security.GeneralSecurityException;
 import java.security.cert.X509Certificate;
+import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Component;
 
+@Profile("!test-signature")
```

Comment:
we have already profiles: `loadtest` and `test` - do we need another one?
are we going to test only the signature here? or do you plan to also test other e2e-scenarios? then I would suggest to use `e2e-test` maybe?

Reply:
I will reuse the one called test, as it is already enabled in INT environment